.github/workflows/template/build-{{ostype}}.yaml.j2

```diff
@@ -620,6 +620,7 @@ jobs:
           {% endif %}
           git config --global user.email "test@github.land"
           git config --global user.name "GitHub Almighty"
+          git config --global datalad.log.exc "1"
```

Comment:
@bpoldrack - this would be the wrong place to "fix" for this new behavior. If we follow the trend we would need to fix any other place (datalad debian and conda-forge packages) similarly... e.g. https://github.com/datalad/datalad/blob/master/datalad/__init__.py#L147 global datalad fixture which establishes config to be used during the tests.  But then ideally there should be some test which tests while explicitly not setting it up

Reply:
note though that at https://github.com/datalad/datalad/blob/master/datalad/__init__.py#L134 (ad-hoc for debian package iirc) we would then need to export config variable I guess

Reply:
> If we follow the trend we would need to fix any other place (datalad debian and conda-forge packages) similarly... 

There's an alternative: https://github.com/datalad/datalad/pull/5691 included the setting in the `datalad test` command. Workflows like the ones here could simply run the tests that way.

However, your suggestion is an option, of course. I'm not sure what you mean by this, though:

> But then ideally there should be some test which tests while explicitly not setting it up

What exactly should such a test assert? Testing that some tests fail when run w/o the config? Is that anyhow more informative than, you know, the failing tests?

Reply:
> There's an alternative: [datalad/datalad#5691](https://github.com/datalad/datalad/pull/5691) included the setting in the `datalad test` command. Workflows like the ones here could simply run the tests that way.

sorry I haven't given sufficient analysis to that PR.  I think we should not tune up all appveyor etc for that, and also not pester `datalad test` which is more of a user-facing interface which is also underused and might got away.  I should have recommended there to move into the `setup_package` fixture.

> > But then ideally there should be some test which tests while explicitly not setting it up
> 
> What exactly should such a test assert?

I was thinking about "not crashing entire datalad process if some log line uses it but no config variable is set".  So pretty much just making sure that `exc_str` or whatever is used not crashing if no config var set.  May be it is already in place, didn't analyze